### ISSUE TITLE
Unable to reset password

### ISSUE THREAD

**Issue by NFarrington** [VATSIM-UK/core#354]
In gitlab by @A-Lawrence on Jan 13, 2016, 17:32

When clicking forgotten password, an error is thrown.

```
[2016-01-13 17:29:36] production.ERROR: exception 'ErrorException' with message 'Argument 1 passed to App\Jobs\Mship\Security\TriggerPasswordReset::__construct() must be an instance of App\Models\Sys\Token, instance of App\Models\Mship\Account given, called in /var/www/core_beta/app/Http/Controllers/Mship/Security.php on line 173 and defined' in /var/www/core_beta/app/Jobs/Mship/Security/TriggerPasswordReset.php:20
```

**Comment by NFarrington** [comment]
In gitlab by @A-Lawrence on Jan 13, 2016, 17:38

mentioned in commit 1eaab5c5e0dbbe3a33a0edde3d66de5146e87df7

**Comment by NFarrington** [comment]
In gitlab by @A-Lawrence on Jan 13, 2016, 18:03

Resolved in commit 1eaab5c5
